draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -323,10 +326,11 @@ and can be sent in a trailer section.
 The `RateLimit-Remaining` response field indicates the remaining `quota-units` defined in {{request-quota}}
 associated to the client.
 
-The header value is
+`RateLimit-Remaining` is an Item {{SF}}.
+Its value MUST be a non-negative Integer.
 
-~~~
-   RateLimit-Remaining = quota-units
```

Comment:
@reschke @ucarion  as `quota-units` is a recurring element, can we define it in terms of `sf-integer` and then say something like:

```
RateLimit-Remaining is an Item {{SF}}:

~~~ abnf
   RateLimit-Remainig = sf-item
~~~

Its value is defined by `quota-units`...
```

Reply:
As an alternative approach, could we instead SFV-ify everything and then have "quota units" be a prosaic concept, rather than an ABNF rule?

Reply:
My mistake, fix it. Should probably fix the referred file under http-extensions